inc/trilogy/builder.h

```diff
@@ -155,7 +173,23 @@ int trilogy_builder_write_lenenc_buffer(trilogy_builder_t *builder, const void *
  * Return values:
  *   TRILOGY_OK     - The value was appended to the packet buffer.
  *   TRILOGY_SYSERR - A system error occurred, check errno.
+ *   TRILOGY_MAX_PACKET_EXCEEDED - Appending this value would exceed the maximum
+ *                              packet size.
  */
 int trilogy_builder_write_string(trilogy_builder_t *builder, const char *data);
 
+/* trilogy_builder_set_max_packet_length - Set the maximum packet length for
+ * the builder. Writing data to the builder that would cause the packet length
+ * to exceed this value will cause the builder to error.
+ *
+ * builder    - A pre-initialized trilogy_builder_t pointer
+ * max_length - The new maximum packet length to set
+ *
+ * Return values:
+ *   TRILOGY_OK     - The maximum packet length was set.
```

Comment:
Very minor nit that you don't really have to do, but would you mind indenting from `-` to line up with the one below, like the rest of the comments? I never used to do stuff like this, but `gofmt` changed my life for better or worse 😆 

I guess the rest of the comments above where `TRILOGY_MAX_PACKET_EXCEEDED` was added could use it as well so everything lines up nicely. But again, no big deal if that's annoying ;)

Reply:
Oh, yeah, this is a bit of an ugly floater. I deliberately didn't want to give up that much horizontal space by aligning everything to the new very-long name, but now this line is only aligned with the separate preceding blocks. :confused:

I see I was also needlessly careless on the wrapped indentation below -- I copied across the misalignment from `buffer.h`, without noticing this is the first instance in this file.

I think we just need to do a bit of a general sweep... I think the alignment was a lot more consistent before one of the renames.

Reply:
I fixed this one, and tidied the wrapping on the others, but couldn't bring myself to drag the other descriptions over that far. 🤷🏻‍♂️